modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding on user-provisioned infrastructure in AWS
+
+You can use forwarding with OpenShift DNS to resolve name queries for your other internal devices using the DNS servers in your data center.
+
+[NOTE]
+====
+The DNS Operator prepends the configuration of `Servers` to `configmap/dns-default`, instead of the entire ConfigMap being statically defined.
```

Comment:
I'm not sure what this note is saying.  Is this referring to falling back to the name servers in `/etc/resolv.conf`?